plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/StandardLanguage.xtend

```diff
@@ -0,0 +1,152 @@
+/*******************************************************************************
+ * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.xtext.generator
+
+import com.google.inject.Injector
+import java.util.ArrayList
+import org.eclipse.xtend.lib.annotations.Accessors
+import org.eclipse.xtext.xtext.generator.builder.BuilderIntegrationFragment2
+import org.eclipse.xtext.xtext.generator.ecore.EMFGeneratorFragment2
+import org.eclipse.xtext.xtext.generator.exporting.QualifiedNamesFragment2
+import org.eclipse.xtext.xtext.generator.formatting.Formatter2Fragment2
+import org.eclipse.xtext.xtext.generator.generator.GeneratorFragment2
+import org.eclipse.xtext.xtext.generator.grammarAccess.GrammarAccessFragment2
+import org.eclipse.xtext.xtext.generator.idea.IdeaPluginGenerator
+import org.eclipse.xtext.xtext.generator.idea.parser.antlr.XtextAntlrIDEAGeneratorFragment
+import org.eclipse.xtext.xtext.generator.junit.Junit4Fragment2
+import org.eclipse.xtext.xtext.generator.parser.antlr.XtextAntlrGeneratorFragment2
+import org.eclipse.xtext.xtext.generator.resourceFactory.ResourceFactoryFragment2
+import org.eclipse.xtext.xtext.generator.scoping.ImportNamespacesScopingFragment2
+import org.eclipse.xtext.xtext.generator.serializer.SerializerFragment2
+import org.eclipse.xtext.xtext.generator.types.TypesGeneratorFragment2
+import org.eclipse.xtext.xtext.generator.ui.compare.CompareFragment2
+import org.eclipse.xtext.xtext.generator.ui.contentAssist.ContentAssistFragment2
+import org.eclipse.xtext.xtext.generator.ui.labeling.LabelProviderFragment2
+import org.eclipse.xtext.xtext.generator.ui.outline.OutlineTreeProviderFragment2
+import org.eclipse.xtext.xtext.generator.ui.outline.QuickOutlineFragment2
+import org.eclipse.xtext.xtext.generator.ui.quickfix.QuickfixProviderFragment2
+import org.eclipse.xtext.xtext.generator.ui.refactoring.RefactorElementNameFragment2
+import org.eclipse.xtext.xtext.generator.ui.templates.CodetemplatesGeneratorFragment2
+import org.eclipse.xtext.xtext.generator.validation.ValidatorFragment2
+import org.eclipse.xtext.xtext.generator.web.WebIntegrationFragment
+import org.eclipse.xtext.xtext.generator.xbase.XbaseGeneratorFragment2
+import org.eclipse.xtext.xtext.generator.xbase.XtypeGeneratorFragment2
+import org.eclipse.xtext.util.internal.Log
+
+/**
+ * @author Sven Efftinge - Initial contribution and API
+ */
+@Accessors(PUBLIC_SETTER)
```

Comment:
The getters are intentionally left out, so I get compiler warnings for non referenced fields :-)
I don't think this class should be designed for extendability, the super class is.